packages/openai.dark

```diff
@@ -194,8 +194,9 @@ module OpenAI =
       | Ok reqBody ->
         let headers = PACKAGE.OpenAI.Config.getHeaders ()
 
+        //CLEANUP: Use HttpClient.request instead of CLI.HttpClient.request once we have the httpclient sorted out
         let openAIResponse =
-          HttpClient.request
+          CLI.HttpClient.request
```

Comment:
None of the following solution I've tried seemed good.
- When moving the endpoint to the dark-editor canvas, we have to load the canvas each time, and we get an`OpenAI API request failed: Request timed out` error most of the times.

- The renaming of `CLI.HttpClient.request` to `HttpClient.request` resulted in the failure of 62 tests. I wasn't sure about commenting all of these out.

The solution I opted for involved using the `CLI.HttpClient.request` function within the `completion` OpenAI package function. This approach ensures that we avoid encountering the `HttpClient.request not found` error.

wdyt?

Reply:
I just merged a fix so you can just use `HttpClient.request`. Let me know if that works for you?

Reply:
it seems to be working well, thanks!